### PR TITLE
feature/gat 4585

### PR DESCRIPTION
**Pull request by camilner-hdr**
Implementation of the new "Making an authenticated request" page

### CODE REVIEW

**Review by dnhdruk — APPROVED** [review]

File: docs/using-our-apis/making-an-authenticated-request.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    }
+    ```
+
+=== " Go "
+
+    ``` go
+    package main
+
+    import {
+        "ftm"
```
Comment (loki-sinclair-hdruk) [inline docs/using-our-apis/making-an-authenticated-request.md:89]: Should be `fmt`

File: docs/using-our-apis/making-an-authenticated-request.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+    ``` cpp
+    #include <iostream>
+    #include <curl/curl.h>
+
+    int main() {
+        CURL *curl;
+        CURLcode res;
+
+        curl = surl_easy_init();
```
Comment (loki-sinclair-hdruk) [inline docs/using-our-apis/making-an-authenticated-request.md:65]: Probs needs to be `curl_easy_init()` ?

File: docs/using-our-apis/making-an-authenticated-request.md (hunk 1 above)
Comment (camilner-hdr) [reply docs/using-our-apis/making-an-authenticated-request.md]: fixed

File: docs/using-our-apis/making-an-authenticated-request.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            painc(err)
+        }
+        defer resp.Body.Close()
+
+        body, err := ioutil.ReadAll(resp.body)
+        if err != nil {
+            panic(err)
+        }
+
+        ftm.Println(string(body))
```
[the remark this answers is not in the document]
Comment (camilner-hdr) [reply docs/using-our-apis/making-an-authenticated-request.md]: fixed

File: docs/using-our-apis/making-an-authenticated-request.md (hunk 2 above)
Comment (camilner-hdr) [reply docs/using-our-apis/making-an-authenticated-request.md]: fixed

**Review by loki-sinclair-hdruk — APPROVED** [review]

**Pull request merged**